src/common/CMakeLists.txt

```diff
@@ -5,32 +5,18 @@
 # SPDX-License-Identifier: (BSD-3-Clause)
 
 set(common_headers
-    boundary_condition.hpp
-    common.hpp
     expr_template_internal.hpp
     expr_template_ops.hpp
-    finite_element_state.hpp
-    initialize.hpp
     serac_types.hpp
```

Comment:
This is the only one left that doesn't fit in the `math` category. Maybe this should also be `infrastructure`?

Reply:
Right now it only contains types used for configuring the physics solvers, so it may be a better fit for `physics_utilities` (previously `problem_description`)